drivers/flash/soc_flash_nrf.c

```diff
@@ -455,11 +479,27 @@ static int erase_op(void *context)
 	}
 #endif
 
+#if defined(CONFIG_SOC_FLASH_NRF_RADIO_SYNC) || \
+    defined(CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE)
```

Comment:
You need to remove this `#if`, otherwise it won't build when none of the two option is enabled.

Reply:
Opps.